src/js/components/Navigation/FooterMainWeVote.jsx

```diff
@@ -172,6 +172,30 @@ class FooterMainWeVote extends Component {
                     trackingOn
                     body={(<span>Credits &amp; Thanks</span>)}
                     className={classes.link}
+                    onClickFunction={() => {
```

Comment:
Hello @pi-sen the OpenExternalWebSite component sends the dataLayer, so you don't need to add this additional onClickFunction. If you dive into OpenExternalWebSite, you will see a way to accomplish everything you are doing here through the internal implementation and props you can pass in if needed.